pkg/services/dashboards/service/dashboard_service.go

```diff
@@ -31,9 +31,7 @@ var (
 		{Action: dashboards.ActionDashboardsWrite, Scope: dashboards.ScopeFoldersAll},
 	}
 	// DashboardServiceImpl implements the DashboardService interface
-	_ dashboards.DashboardService             = (*DashboardServiceImpl)(nil)
-	_ dashboards.DashboardProvisioningService = (*DashboardServiceImpl)(nil)
-	_ dashboards.PluginService                = (*DashboardServiceImpl)(nil)
```

Comment:
these are still true right?

Reply:
i'd ignore this PR for now while it's in draft. i attempted to revert commits, and that might not be the best approach here, but i'm still trying to sort through the diff